CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.1 (2020-01-08)
+
+- Fixed the file drop component to accept multiple files when drag and dropped. [#92](https://github.com/blackbaud/skyux-forms/pull/92)
```

Comment:
Is the component "file attachment" or "file drop"? 

"...to accept multiple files with drag and drop" or "...to accept multiple files when using drag and drop." 

Just trying to write around "when drag and dropped."

Reply:
Looking at the PR, I'm pretty sure "file drop component," is correct, although the PR also updates the file attachment component and service, so it'd be good if Alex weighs in. As for the rewording, what do you think about `Fixed the file drop component to enable users to drag and drop multiple files.`?

Reply:
Yep that works for me. Sorry, I had a side convo on slack with @Blackbaud-DeniseCisneros , but I essentially said the same thing - it is indeed the file drop component. I'll go ahead and make that change.

Reply:
Done!